src/cljs/proton/layers/lang/typescript/core.cljs

```diff
@@ -0,0 +1,27 @@
+(ns proton.layers.lang.typescript.core
+  (:require [proton.lib.helpers :refer [console!]]
+            [proton.lib.mode :as mode])
+  (:use [proton.layers.base :only [init-layer! init-package get-packages describe-mode register-layer-dependencies]]))
+
+
+(defmethod init-layer! :lang/typescript
+  [_ {:keys [config layers]}]
+  (console! "init" :lang/typescript))
+
+(defmethod get-packages :lang/typescript
+  []
+  [:atom-typescript
+   :linter-tslint])
```

Comment:
oh whoops, looks like I merged too fast. Could you move the `linter-tslint` into `register-layer-dependencies`?